plugin/mutate/source/dextool/plugin/mutate/backend/analyze/package.d

```diff
@@ -1111,23 +1110,59 @@ struct Analyze {
         import std.algorithm : filter;
         import clang.c.Index : CXTokenKind;
         import dextool.plugin.mutate.backend.database : LineMetadata, FileId, LineAttr, NoMut;
+        import std.format;
 
         if (auto localId = file in result.idFile) {
             const fid = FileId(localId.get);
 
+            int sectionStart = -1;
             auto mdata = appender!(LineMetadata[])();
             foreach (t; tstream.getTokens(file).filter!(a => a.kind == CXTokenKind.comment)) {
                 auto m = matchFirst(t.spelling, re_nomut);
-                if (m.whichPattern == 0)
+
+                if(m.whichPattern == 0)
                     continue;
 
-                () @trusted {
-                    mdata.put(LineMetadata(fid, t.loc.line,
+                switch (m[4]){
+                    case "BEGIN":
+                        if(sectionStart == -1){
+                            sectionStart = t.loc.line;
+                        } else {
+                            logger.warning(format!"NOMUT: Found multiple NOMUTBEGIN in a row! Will use the first one, line %d"(sectionStart));
```

Comment:
Use warningf instead of format.
You can also use %s instead of %d because warningf will detect the int and convert it to a string